### DISCUSSION TITLE
Symbol style &

### DISCUSSION THREAD

**Discussion by ustas7700** [tonsky/FiraCode#1572] [General]
I appreciate your work and I like your font, but the **&** symbol looks terrible and doesn't look like itself. Could you change that? Thank you.

**Comment by tonsky** [comment]
ss03 solves that https://github.com/tonsky/FiraCode/wiki/How-to-enable-stylistic-sets

**Reply by ustas7700** [reply]
Is this possible for Visual Studio? Not for VSCode.

**Reply by tonsky** [reply]
That I don’t know

**Reply by ustas7700** [reply]
Okay, JetBrains Mono is not bad either.
